charts/ai-service/templates/artifacts-cache.yaml

```
@@ -0,0 +1,15 @@
+{{- if .Values.artifactsCache.enabled -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.artifactsCache.name | default (include "common.fullname" .) }}
```

potentially dangerous, could have two PVC with same name!
```suggestion
  name: {{ .Values.artifactsCache.name | default (include "common.fullname" .) }}-artifacts
```